src/proposals/Addresses.sol

```diff
@@ -564,6 +564,37 @@ contract Addresses is Test, ChainIds {
             baseChainId,
             0x74Cbb1E8B68dDD13B28684ECA202a351afD45EAa
         );
+
+        /// chainlink eth oracles
+        _addAddress(
+            "CHAINLINK_ETH_USD",
+            baseChainId,
+            0x71041dddad3595F9CEd3DcCFBe3D1F4b0a16Bb70
```

Comment:
Verified this is correct here: https://docs.chain.link/data-feeds/price-feeds/addresses?network=base